tokio/src/net/mod.rs

```diff
@@ -7,14 +7,14 @@
 //!
 //! # Organization
 //!
-//! * [`TcpListener`] and [`TcpStream`] provide functionality for communication over TCP
-//! * [`UdpSocket`] provides functionality for communication over UDP
-//! * [`UnixListener`] and [`UnixStream`] provide functionality for communication over a
+//! [`TcpListener`] and [`TcpStream`] provide functionality for communication over TCP
+//! [`UdpSocket`] provides functionality for communication over UDP
+//! [`UnixListener`] and [`UnixStream`] provide functionality for communication over a
 //! Unix Domain Stream Socket **(available on Unix only)**
-//! * [`UnixDatagram`] provides functionality for communication
+//! [`UnixDatagram`] provides functionality for communication
 //! over Unix Domain Datagram Socket **(available on Unix only)**
-//! * [`tokio::net::unix::pipe`] for FIFO pipes **(available on Unix only)**
-//! * [`tokio::net::windows::named_pipe`] for Named Pipes **(available on Windows only)**
+//! [`tokio::net::unix::pipe`] for FIFO pipes **(available on Unix only)**
+//! [`tokio::net::windows::named_pipe`] for Named Pipes **(available on Windows only)**
```

Comment:
You're removing a lot of lists. Please don't do that.

Reply:
This is a clippy warning.

Reply:
Do not introduce bugs or break our documentation just because clippy warns. In this case, I imagine that you should do this:
```diff
 //! * [`UnixDatagram`] provides functionality for communication
-//! over Unix Domain Datagram Socket **(available on Unix only)**
+//!   over Unix Domain Datagram Socket **(available on Unix only)**
```
instead of removing the list.

Reply:
There are many lists that are wrong in this way. Please fix all of them.